actionpack/test/dispatch/system_testing/system_test_case_test.rb

```diff
@@ -6,6 +6,14 @@ class SetDriverToRackTestTest < DrivenByRackTest
   end
 end
 
+class OverrideDriverToRackTestTest < DrivenBySeleniumWithChrome
```

Comment:
I was confused why this test was necessary when the test before works with `rack_test`. Can you rename it to `OverrideSeleniumSubclassToRackTestTest` so it's clearer?